cylc/flow/scripts/cylc_insert.py

```diff
@@ -74,8 +74,8 @@ def main(parser, options, suite, *items):
 
     pclient(
         'insert_tasks',
-        {'items': items, 'no_check': options.no_check,
-         'stop_point_string': options.stop_point_string},
+        {'tasks': items, 'check_point': not options.no_check,
```

Comment:
Just checking, this is still `options.no_check` right? The value in `options` is kept for backward compatibility, right?

Reply:
Yeah, I haven't changed the CLI to match this interface yet.